vehicle_status_gui:
  ros__parameters:
    log_level: "debug"
    topics: ["empty",]
    frame_rate: 20
